consensus/state.go

```diff
@@ -1955,11 +2047,10 @@ func (cs *State) addVote(
 	switch vote.Type {
 	case tmproto.PrevoteType:
 		prevotes := cs.Votes.Prevotes(vote.Round)
-		cs.Logger.Info("Added to prevote", "vote", vote, "prevotes", prevotes.StringShort())
```

Comment:
We don't want each vote to come in to be info? I guess with 150 validators that can be quite overbearing

Reply:
No, it's a lot. Keep in mind we can get multiple votes from multiple validators/peers multiple times per validator/peer. It can get pretty noisy!